File: SIMS-SPA/src/app/model/course.ts
export interface ICourse {
    uuid: number;
    courseTitle: string;
    courseNum: number;
    section: number;
    scheduleStartTime: string;
    scheduleEndTime: string;
    instructor: string;
    room: string;
    creditHours: number;
    crn: string;
    maxStudent: number;
    notes: string;
    weekday: string;
    scheduleType: number;
    semesterId: number;
}

File: SIMS.API/Models/Course.cs
using System.ComponentModel.DataAnnotations;

namespace SIMS.API.Models
{
    public class Course
    {
        [Key]
        public int uuid { get; set; }
        public string courseTitle { get; set; }
        public string courseNum { get; set; }
        public int section { get; set; }
        public string scheduleStartTime { get; set; }
        public string scheduleEndTime { get; set; }
        public string instructor { get; set; }
        public string room { get; set; }
        public int creditHours { get; set; }
        public string crn { get; set; }
        public int MaxStudent { get; set; }
        public string notes { get; set; }
        public string weekday { get; set; }
        public int scheduleType { get; set; }
        public int semesterId { get; set; }

    }
} 